Sources/SwiftSDKGenerator/Generator/LocalSwiftSDKGenerator.swift

```diff
@@ -40,6 +40,7 @@ public final class LocalSwiftSDKGenerator: SwiftSDKGenerator {
       .removingLastComponent()
       .removingLastComponent()
       .removingLastComponent()
+      .removingLastComponent()
```

Comment:
Fix for `Bundles` ending up in the `Sources` directory after `LocalSwiftSDKGenerator.swift` was moved to a subdirectory.